components/webui/client/src/pages/SearchPage/Presto/ProgressBar/index.module.css

```diff
@@ -1,3 +1,9 @@
+.prestoProgressContainer {
+    /* A `position: relative` container is needed for the `prestoProgress` element to have the correct
+    width when using `position: absolute`. */
+    position: relative;
+}
+
 .prestoProgress {
     position: absolute;
```

Comment:
as discussed offline (and i tested), only changing this to `position: static;` should solve the issue, without causing other visual defects